src/main/java/org/fogbowcloud/arrebol/execution/docker/tasklet/TaskletHelper.java

```diff
@@ -0,0 +1,112 @@
+package org.fogbowcloud.arrebol.execution.docker.tasklet;
+
+import java.util.List;
+import org.apache.log4j.Logger;
+import org.fogbowcloud.arrebol.execution.TaskExecutionResult;
+import org.fogbowcloud.arrebol.execution.docker.DockerCommandExecutor;
+import org.fogbowcloud.arrebol.models.command.Command;
+
+public class TaskletHelper {
+    private static final String taskScriptExecutorFilePath = "/tmp/task-script-executor.sh";
```

Comment:
Why not follow the pattern of the rest of the constants?

Reply:
I mean, using only uppercase

Reply:
Done.